src/05-objects.js

```diff
@@ -118,7 +141,10 @@ function addKeyToObject(object, key) {}
  *  addKeyValuePairToObject({ up: true }, "left", false);
  *  //> { up: true, left: false }
  */
-function addKeyValuePairToObject(object, key, val) {}
+function addKeyValuePairToObject(object, key, val) {
```

Comment:
All of our previous object functions should have indentation like this one.  It makes it much easier to read.